src/basics.md

```diff
@@ -1385,6 +1386,44 @@ fn main() {
 }
 ```
 
+[ex-examine-datetime]: #ex-examine-datetime
+<a name="ex-examine-datetime"></a>
+## Examine a DateTime struct via Datelike and Timelike traits
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Gets the current UTC [`DateTime`] and its hour/minute/second via [`Timelike`]
+and its year/month/day/weekday via [`Datelike`].
+
+```rust
+extern crate chrono;
+use chrono::{Datelike, Timelike, Utc};
+
+fn main() {
+    let now = Utc::now();
+
+    // available due to chrono::Timelike
+    let second = now.second();
```

Comment:
I'd suggest:
- running rustfmt on the example
- dropping the intermediate variables and making the queries directly in the `println!`body
```rust
    let (is_pm, hour) = now.hour12();
    println!(
        "The current UTC time is {:02}:{:02}:{:02} {}",
        hour,
        now.minute(),
        now.second(),
        if is_pm { "PM" } else { "AM" },
    );
```

The example should ges slightly less noisy.

Reply:
I agree. Good change